SteamKit2/SteamKit2/Steam/CDN/Client.cs

```diff
@@ -227,10 +230,24 @@ public async Task<int> DownloadDepotChunkAsync( uint depotId, DepotManifest.Chun
                 }
             }
 
+            if ( !CheckedForLancacheServer )
```

Comment:
I still would prefer this to be an explicit separate call a consumer needs to call.